.github/ISSUE_TEMPLATE/nova_tarefa.md

```
@@ -0,0 +1,40 @@
+### Descrição
+[Descrição detalhada do que precisa ser feito]
+
+### Objetivo
+[Qual o objetivo/resultado esperado desta tarefa]
+
+### Critérios de Aceitação
+- [ ] Critério 1
+- [ ] Critério 2
+- [ ] Critério 3
+
+### Informações Técnicas
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)